modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/controller/AccountController.java

```diff
@@ -92,4 +97,20 @@ public AccountController(AuthenticationManager authMgr,  AccountsService account
 
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * @param curUser Current user.
+     * @param changes Changes to apply to user.
+     * @return {@linkplain HttpStatus#OK OK} on success.
+     */
+    @PostMapping(path = "/api/v1/profile/save", consumes = APPLICATION_JSON_VALUE)
+    public ResponseEntity<Void> save(@AuthenticationPrincipal Account curUser, @RequestBody JsonObject changes) {
```

Comment:
Why we use JsonObject instead of POJO class?

Reply:
Fixed.